prusti-viper/src/encoder/snapshot/mod.rs

```diff
@@ -0,0 +1,228 @@
+use std::collections::HashMap;
+
+use log::{info, warn};
+use prusti_common::vir;
+
+pub use self::purifier::ExprPurifier;
+
+use super::{errors::EncodingResult, snapshot_encoder::Snapshot};
+use vir::Type;
```

Comment:
Why this type specifically? The file uses `vir::Type` in many (most?) places anyway.

Reply:
Removed the type.